### ISSUE TITLE
Additional CSS in the Head tag which breaks theme CSS

### ISSUE THREAD

**Issue by astab** [TryGhost/Ghost#15020]
### Issue Summary

After upgrade to Ghost 5.3.0 I see that additional CSS has been added in the Head tag which breaks the theme style:

```
<style>@tailwind base;
@tailwind components;
@tailwind utilities;
/* Custom reset */
html {
    font-size: 62.5%;
}
body {
    font-size: 1.5rem;
}
.ghost-display {
    display: block !important;
}
</style>
```
The demo of the theme I use: https://scriptor.justgoodthemes.com/

### Steps to Reproduce

1. Update to the latest Ghost version 5.3.0.
2. Use custom theme.

### Ghost Version

5.3.0

### Node.js Version

v14.19.0

### How did you install Ghost?

Digital Ocean

### Database type

MySQL 8

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by VeloAddict** [comment]
```
"sodoSearch": {
    "url": ""
  }
```
add this to your configuration and restart ghost

**Comment by rishabhgrg** [comment]
@astab Thanks for reporting. This has been now patched and released in the bundled search script which was released in Ghost 5.3. The fix is auto applied on next page refresh, no action/change is needed from your side.
